roles/common.nix

```diff
@@ -30,9 +30,6 @@
   # Just disable it since we are using telegraf to monitor raid health.
   systemd.services.mdmonitor.enable = false;
 
-  # enable "sar" system activity collection
-  services.sysstat.enable = true;
```

Comment:
Not sure if anyone is looking at the output of this, doesn't look like telegraf collects it?

Reply:
I think so.